apps/fabric-website/src/styles/_variables.scss

```diff
@@ -61,7 +61,7 @@ $Header-backgroundColor: $ms-color-black;
 $Header-height: 50px;
 $Header-height-full-uhf: $Header-height * 2;
 
-$Nav-width: 208px;
+$Nav-width: 230px;
```

Comment:
@Jahnp - this seems like it would have an impact on the layout of the site itself. Can you approve this as well?

Reply:
Yeah, the nav width probably shouldn't change until I port over the internal site redesign.

Reply:
okay, I'll change it back and reduce the width of the searchbox